src/templates/getHandler.ts

```diff
@@ -41,39 +43,48 @@ const makeHandler =
 
     augmentFsModule({ promises, staticManifest, pageRoot, getBase: () => base })
 
-    const NextServer = getNextServer()
-
-    const nextServer = new NextServer({
-      conf,
-      dir: path.resolve(__dirname, app),
-      customServer: false,
-    })
-    const requestHandler = nextServer.getRequestHandler()
-    const server = new Server(async (req, res) => {
-      try {
-        await requestHandler(req, res)
-      } catch (error) {
-        console.error(error)
-        throw new Error('server function error')
+    let bridge
```

Comment:
```suggestion
    let bridge: Bridge
```

? I'm guessing it's possibly not an available type from Vercel